lib/assets/core/javascripts/cartodb3/editor/export-image-pane/export-image-pane.js

```
@@ -61,6 +67,15 @@ module.exports = CoreView.extend({
     });
   },
 
+  _addNotification: function (error) {
```

Do we hide the error notification if the user tries again the export?